local.yml

```diff
@@ -21,6 +21,8 @@ services:
     ports:
       - '8000:8000'
     command: /start
+    stdin_open: true
+    tty: true
```

Comment:
I assume this is to support the attach command you added? Is this only for testing on local?

Reply:
Handy for debugging as you can add a break and get into pdb